src/plugins/explorer/queue.js

```diff
@@ -1,42 +1,51 @@
-'use strict';
+'use strict'
 
-const { debounce, groupBy, merge, noop, reduce } = require('lodash');
-const debug = require('debug')('lmr-wallet:core:explorer:queue');
-const getTransactionStatus = require('./transaction-status');
-const promiseAllProps = require('promise-all-props');
-const LumerinContracts = require('@lumerin/contracts')
+const { debounce, groupBy, merge, noop, reduce } = require('lodash')
+const debug = require('debug')('lmr-wallet:core:explorer:queue')
+const getTransactionStatus = require('./transaction-status')
+const promiseAllProps = require('promise-all-props')
 
-function createQueue (config, eventBus, web3) {
-  debug.enabled = config.debug;
+const erc20TransferABI = [
```

Comment:
`const abi = require('./erc20-abi.json');`

is it the same code?

Reply:
changed code to use this abi file